test/e2e/server.ts

```diff
@@ -0,0 +1,95 @@
+import { ChildProcess, fork } from 'node:child_process';
+import path from 'node:path';
+import readline from 'node:readline';
+import fs from 'node:fs';
+import ts from 'typescript/lib/tsserverlibrary';
+
+type Command = `${ts.server.protocol.CommandTypes}`;
```

Comment:
Why are we interpolating this?

Reply:
This basically converts the `CommandTypes` enum to a union, I found that more ergonomic to use with `sendCommand`. You can just use a string instead of importing the enum